components/image/slider/src/index.js

```diff
@@ -9,6 +9,10 @@ export const IMAGE_SLIDER_COUNTER_POSITIONS = {
   BOTTOM_LEFT: 'bottomLeft',
   BOTTOM_RIGHT: 'bottomRight'
 }
+export const IMAGE_SLIDER_IMAGE_TYPES = {
+  SRC: 'src',
+  CARD: 'card'
```

Comment:
IMHO it'd make more sense to me to name the default type `img` as it ends up rendering an image, and whatever you can pass it on, the other type, could be just a `node`... what do you think?

```suggestion
  IMG: 'img',
  NODE: 'node'
```